problems/2.functions.js

```diff
@@ -20,16 +21,22 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
-
+function incrementGlobalCount(val) {
+  
 }
+
 /** Multiplies three numbers
 * @param {number} firstNum - The first number
 * @param {number} secondNum - The second number
 * @param {number} thirdNum - The third number
 * @returns {number} - The product of the three numbers
 */
-function multiply() {
+function multiply(firstNum, secondNum , thirdNum) {
+  let num1 = 1
+  let num2 = 2
+  let num3 = 3
```

Comment:
Why these numbers? You want to sue the numbers that are being passed in as arguments. Do NOT hardcode your own.